packages/components/src/font-size-picker/index.js

```diff
@@ -21,14 +21,19 @@ function getSelectValueFromFontSize( fontSizes, value ) {
 	return 'normal';
 }
 
-function getSelectOptions( optionsArray ) {
+function getSelectOptions( optionsArray, disableCustomFontSizes ) {
+	if ( ! disableCustomFontSizes ) {
+		optionsArray = [
+			...optionsArray,
+			{ slug: 'custom', name: __( 'Custom' ) },
+		];
+	}
 	return [
```

Comment:
The return statement is over-complicated since you can just return `optionsArray.map(...)` directly. There's no need to spread the array into another array.